components/catalogs/front/src/components/ProductMapping/hooks/useInfiniteSourceAttributes.unit.ts

```diff
@@ -55,7 +55,7 @@ test('it fetches attributes & paginates', async () => {
     );
 
     expect(fetchMock).toHaveBeenCalledWith(
-        '/rest/catalogs/attributes-by-target-type-and-target-format?page=1&limit=2&search=&targetType=string&targetFormat=uri',
+        '/rest/catalogs/attributes_by_target_type_and_target_format?page=1&limit=2&search=&targetType=string&targetFormat=uri',
```

Comment:
```suggestion
        '/rest/catalogs/attributes_by_target_type_and_target_format?page=1&limit=2&search=&targetType=string&targetFormat=uri',
```

@masacc what's the correct value here ?

Reply:
now it's kebab-case (Rifat is the new black)